builtin/settingtypes.txt

```diff
@@ -553,21 +551,6 @@ shadow_filters (Shadow filter quality) enum 1 0,1,2
 #    Requires: enable_dynamic_shadows, opengl
 shadow_map_color (Colored shadows) bool false
 
-#    Spread a complete update of shadow map over given number of frames.
-#    Higher values might make shadows laggy, lower values
-#    will consume more resources.
-#    Minimum value: 1; maximum value: 16
-#
-#    Requires: enable_dynamic_shadows, opengl
-shadow_update_frames (Map shadows update frames) int 8 1 16
-
-#    Set the soft shadow radius size.
-#    Lower values mean sharper shadows, bigger values mean softer shadows.
-#    Minimum value: 1.0; maximum value: 15.0
-#
-#    Requires: enable_dynamic_shadows, opengl
-shadow_soft_radius (Soft shadow radius) float 5.0 1.0 15.0
```

Comment:
Soft radius does not strike me as an advanced setting.

Reply:
moved it back.